ext/mini_racer_extension/extconf.rb

```diff
@@ -1,3 +1,11 @@
+if defined?(TruffleRuby)
```

Comment:
`RUBY_ENGINE == "truffleruby"` should always be preferred in non-truffleruby-internal code.

Reply:
FIxed